daemon/src/wallet.rs

```diff
@@ -354,17 +356,70 @@ where
     }
 }
 
+struct LockedUtxos {
+    inner: HashSet<(Instant, OutPoint)>,
+    time_to_lock: Duration,
+}
+
+impl LockedUtxos {
+    fn new(network: Network) -> Self {
+        let time_to_lock = match network {
+            Network::Bitcoin => Duration::from_secs(20 * 60),
```

Comment:
❓ At the moment we only lock the UTXOs when creating a new DLC, i.e. during contract setup (but not rollover because we use the original lock). This is irrelevant for all spending transactions from lock.
Is what I am stating correct?

Reply:
Yes, you're right! We only use wallet UTXOs for the lock transaction, which is only constructed during contract setup.